test/parallel/test-performance-eventlooputil.js

```diff
@@ -23,28 +23,28 @@ if (nodeTiming.loopStart === -1) {
 }
 
 setTimeout(mustCall(function r() {
-  const t = Date.now();
   const elu1 = eventLoopUtilization();
```

Comment:
I messed up here. `elu1` wasn't also shouldn't be captured until after some `idleTime` has accumulated. This function call was only meant supposed to check if `.active` was being set correctly.

The reason for forcing `idleTime` to accumulate before the test ran was so it would execute the same running it directly or running with `./tools/test.py --worker`.